introduction.md

```diff
@@ -1,2 +1,10 @@
 Hi, my username is KevinDyer. I need to make this a longer message so I am...derp!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?